### PR TITLE
Display use of istioctl bug-report command

### PR DESCRIPTION
**Pull request by shamsher31**
More details in https://github.com/istio/istio/pull/29148 and https://github.com/istio/istio.io/pull/8573

### CODE REVIEW

**Comment by shamsher31** [comment]
@ericvn The PR fails because of `vulnerability error`
https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/29185/release-test_istio/1331495829936017408

**Review by mandarjog — COMMENTED** [review]
Instead of removing script, can you replace the content with “echo use istioctl bug-report; exit 1” or equivalent?

We will remove the file in the next release.

**Comment by shamsher31** [comment]
> Instead of removing script, can you replace the content with “echo use istioctl bug-report; exit 1” or equivalent?

Sounds good.

File: tools/dump_kubernetes.sh [hunk 1]
```diff
@@ -14,358 +14,8 @@
...
-  return ${exit_code}
-}
-
-stored_log_bytes=0
+set -e
 
-main "$@"
+echo "Use istioctl bug-report instead.";
+echo "https://istio.io/latest/about/bugs/#generating-a-cluster-state-archive";
+exit 0;
```
Comment (mandarjog) [inline tools/dump_kubernetes.sh:21]: exit 1

The script should error out.

**Review by mandarjog — APPROVED** [review]
Lgtm

Please remove do-not-merge after changing return value.

File: tools/dump_kubernetes.sh (hunk 1 above)
Comment (shamsher31) [reply tools/dump_kubernetes.sh]: Done. Thanks.

**Review by howardjohn — COMMENTED** [review]
LGTM

**Pull request merged**